### DISCUSSION TITLE
How to expand diff of multiple/all files in status?

### DISCUSSION THREAD

**Discussion by sQVe** [NeogitOrg/neogit#1210] [Q&A] [answered]
Hey, 👋

I often stage hunks in files and I'm finding it tedious to open the diff for each file in the status window. Is there a way to expand or close all of them, like in Fugitive?

🍻

**Comment by CKolkey** [comment, marked as answer]
You can use `1`, `2`, `3`, and `4` to expand/collapse everything.
